### PR TITLE
OI

### PR DESCRIPTION
**Pull request by Imperatoras**
I've successfully populated the OI.java file with control methods, according to the preferences of Coby/Bryant. My code is fully up to date with master and ready to be auto-merged :D

### CODE REVIEW

File: src/main/java/frc/robot/Robot.java [hunk 1]
```diff
@@ -32,6 +34,7 @@
     public static Floop floop;
     public static Compressor compressor;
     public static Rollers rollers;
+	public static Fangs fangs;
```
Comment (justinkim) [inline src/main/java/frc/robot/Robot.java]: Weird indentation

File: src/main/java/frc/robot/RobotMap.java [hunk 2]
```diff
@@ -9,9 +9,12 @@
 
 public interface RobotMap {
 
+    /**
```
Comment (justinkim) [inline src/main/java/frc/robot/RobotMap.java]: There's an empty comment block

File: src/main/java/frc/robot/RobotMap.java [hunk 3]
```diff
@@ -46,22 +49,23 @@
     int RIGHT_BOTTOM_MOTOR_PORT = -1;
   
     double WHEEL_DIAMETER = 6;
-    double WHEEL_INCHES_PER_REVOLLUTION = WHEEL_DIAMETER*Math.PI;
+    double WHEEL_INCHES_PER_REVOLUTION = WHEEL_DIAMETER*Math.PI;
```
Comment (justinkim) [inline src/main/java/frc/robot/RobotMap.java]: Put spaces around the `*`

File: src/main/java/frc/robot/subsystems/Fangs.java [hunk 4]
```diff
@@ -31,18 +28,13 @@ public void initDefaultCommand() {
         // setDefaultCommand(new MySpecialCommand());
     }
 
-    @Override
-    public void periodic() {
```
Comment (justinkim) [inline src/main/java/frc/robot/subsystems/Fangs.java]: It's okay to leave this method here

File: src/main/java/frc/robot/subsystems/Rollers.java [hunk 5]
```diff
@@ -30,6 +30,10 @@ public void deacquire() {
         setSpeed(-1);
     }
 
+    public void acquireSpeed(double speed) {
```
Comment (justinkim) [inline src/main/java/frc/robot/subsystems/Rollers.java]: This method is needless, you can simply use `setSpeed()` itself.

File: src/main/java/frc/robot/Robot.java (hunk 1 above)
Comment (Imperatoras) [reply src/main/java/frc/robot/Robot.java]: I don't see that in my version of VSCode...

Comment (justinkim) [reply src/main/java/frc/robot/Robot.java]: Be sure that your workspace indentation settings are set to 4 spaces

File: src/main/java/frc/robot/RobotMap.java (hunk 3 above)
Comment (Imperatoras) [reply src/main/java/frc/robot/RobotMap.java]: this wasn't me in the first place but I'll fix it

File: src/main/java/frc/robot/Robot.java (hunk 1 above)
Comment (Imperatoras) [reply src/main/java/frc/robot/Robot.java]: They are .-.

File: src/main/java/frc/robot/subsystems/Fangs.java [hunk 6]
```diff
@@ -31,18 +28,17 @@ public void initDefaultCommand() {
...
-        super.periodic();
-    }
+    public void periodic(){
```
Comment (justinkim) [inline src/main/java/frc/robot/subsystems/Fangs.java]: Don't forget to restore the `@Override` annotation and the call to `super.periodic()`

**Pull request merged**